graspy/cluster/gclust.py

```diff
@@ -56,10 +67,16 @@ class GaussianCluster(BaseCluster):
         all possible number of clusters given by range(1, max_components).
     """
 
-    def __init__(self, max_components=1, covariance_type="full", random_state=None):
+    def __init__(
+        self,
+        max_components=1,
+        min_components=1,
```

Comment:
Switch order of min and max_components. I think it makes sense to go smaller to larger.